database/PostgreSQL/queries.js

```diff
@@ -0,0 +1,24 @@
+const { Pool } = require('pg');
+const PASSWORD = require('../passwordIgnore.js');
+
+const pool = new Pool({
+  user: 'postgres',
+  host: 'localhost',
+  database: 'images',
+  password: PASSWORD.PASSWORD,
+  port: 5432
+});
```

Comment:
Rather than creating a new pool variable here, could this be replaced with require('./database.js') since you are exporting pool from there?

Reply:
Very true. I will fix this in a later version.